2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Configure stats pid to 20512
2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Loading settings from C:\Users\idris\.config\wandb\settings
2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Loading settings from C:\Users\idris\Desktop\Projects\MLOps-Journey\week_2_hydra_config\outputs\2024-10-07\19-02-48\wandb\settings
2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-07 19:02:52,568 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'week_2_hydra_config\\train.py', 'program_abspath': 'C:\\Users\\idris\\Desktop\\Projects\\MLOps-Journey\\week_2_hydra_config\\train.py', 'program': 'C:\\Users\\idris\\Desktop\\Projects\\MLOps-Journey\\week_2_hydra_config\\train.py'}
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_init.py:_log_setup():532] Logging user logs to .\wandb\run-20241007_190252-l32ea856\logs\debug.log
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_init.py:_log_setup():533] Logging internal logs to .\wandb\run-20241007_190252-l32ea856\logs\debug-internal.log
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_init.py:init():617] calling init triggers
2024-10-07 19:02:52,569 INFO    MainThread:20512 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-07 19:02:52,570 INFO    MainThread:20512 [wandb_init.py:init():667] starting backend
2024-10-07 19:02:52,570 INFO    MainThread:20512 [wandb_init.py:init():671] sending inform_init request
2024-10-07 19:02:52,577 INFO    MainThread:20512 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2024-10-07 19:02:52,578 INFO    MainThread:20512 [wandb_init.py:init():684] backend started and connected
2024-10-07 19:02:52,592 INFO    MainThread:20512 [wandb_init.py:init():779] updated telemetry
2024-10-07 19:02:52,892 INFO    MainThread:20512 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-07 19:02:53,022 ERROR   MainThread:20512 [wandb_init.py:init():838] encountered error: failed to upsert bucket: returned error 404 Not Found: {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-10-07 19:02:53,022 ERROR   MainThread:20512 [wandb_init.py:init():1262] error in wandb.init()
Traceback (most recent call last):
  File "C:\Users\idris\.conda\envs\mlops_journey\Lib\site-packages\wandb\sdk\wandb_init.py", line 1252, in init
    return wi.init()
           ^^^^^^^^^
  File "C:\Users\idris\.conda\envs\mlops_journey\Lib\site-packages\wandb\sdk\wandb_init.py", line 844, in init
    raise error
wandb.errors.errors.CommError: failed to upsert bucket: returned error 404 Not Found: {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-10-07 19:02:53,812 WARNING MsgRouterThr:20512 [router.py:message_loop():77] message_loop has been closed
